### PR TITLE
Document paper endpoints

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: docs/hub/api.md [hunk 1]
```diff
@@ -258,6 +259,18 @@ headers = { "authorization" :  "Bearer $token" }
 
 This is equivalent to `huggingface_hub.whoami()`.
 
+## Paper Pages API
+
+The following endpoint gets information about a paper.
+
+### GET /api/arxiv/{arxiv_id}/repos
+
+Get all the models, datasets, and Spaces mentioned in a paper.
```
Comment (julien-c) [inline docs/hub/api.md:268]: we won't be able to use the hits for this to represent actual calls from Arxiv 

but i guess that's fine?

Comment (osanseviero) [reply docs/hub/api.md:268]: From my side it's fine

Comment (julien-c) [reply docs/hub/api.md:268]: BTW this is not really what this is doing

```suggestion
### GET /api/arxiv/{arxiv_id}/repos

Get all the models, datasets, and Spaces that refer to a paper.
```

better?^

**Review by julien-c — APPROVED** [review]
ok for me with my suggestion

**Review by SBrandeis — APPROVED** [review]

**Pull request merged**